src/components/input/SelectNext.tsx

```diff
@@ -273,21 +336,22 @@ function SelectMain<T>({
       </button>
       <SelectContext.Provider value={{ selectValue, value }}>
         <ul
+          // @ts-expect-error Popover attribute not yet part of type
+          popover
```

Comment:
The `popover attribute is missing in preact's JSX types. We discussed defining our custom type, but I ended up just using `@ts-expect-error`, as it's more straightforward and easier to remove once this has been fixed upstream.

We have also used a similar approach for some other attributes, like in [`Slider`](https://github.com/hypothesis/frontend-shared/blob/3d2b282325399cd65276ceb1a4107070c7b078a6/src/components/transition/Slider.tsx#L84)